### PR TITLE
Billing information

### LINKED ISSUE #940: Migrate article: Tax information regarding Aiven services
# What should we add to our content collections?

Migrate the content from this article to fit the documentation portal format https://help.aiven.io/en/articles/1484583-tax-information-regarding-aiven-services

# Is this something you would work on yourself?

[x] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by ewe-gra**
I am adding the article with Tax Information from help center https://help.aiven.io/en/articles/1484583-tax-information-regarding-aiven-services. 
 
Fixes #940

### CODE REVIEW

File: docs/platform/concepts/tax-information.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+The value-added tax percentage depends on the domicile of the customer.
+
+For business customers in EU countries, other than Finland, we can apply the reverse charge mechanism of 2006/112/EC article 196, 
+when a valid VAT ID is entered into the billing information of the relevant Aiven project.
+
+Rest of the world
+-----------------
+
+No value-added tax is applied to services sold outside the European Union as described in Value-Added Tax Act of Finland section 69h. 
+Finland has tax treaties with most countries in the world affirming this status.
```
Comment (dewan-ahmed) [inline docs/platform/concepts/tax-information.rst:21]: Finland has tax treaties with most countries in the world, affirming this status. (comma added)

**Review by dewan-ahmed — COMMENTED** [review]
Could you please add an entry for billing under the file concepts.rst as well?

I understand that you're already working on fixing the step by step section. 

Thanks for this PR.

**Comment by lornajane** [comment]
The concepts page is this one https://developer.aiven.io/docs/platform/concepts

File: docs/platform/concepts/tax-information.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+United States
+-------------
+
+According to the tax treaty between Finland and United States no tax needs to be withhold from payments by US entities to Aiven.
```
[the remark this answers is not in the document]
Comment (ewe-gra) [reply docs/platform/concepts/tax-information.rst]: Solved it.

**Comment by ewe-gra** [comment]
@dewan-ahmed , I applied the fixes, it's ready for review.

**Comment by laysauchoa** [comment]
we have a lot of small commits, maybe is good to squash before merging.

**Comment by lornajane** [comment]
Could be good to squash (meaning to combine the work into fewer separate commits) but given that this is a PR from a new contributor, I'd expect the reviewer to handle it.

**Comment by lornajane** [comment]
Using Dewan's rebased version of this PR, if the build passes I think we are in good shape now :)

**Review by dewan-ahmed — APPROVED** [review]
LGTM

Great work @ewe-gra 🎉 

Thanks @laysauchoa & @lornajane for your review/help 🙏🏽

**Pull request merged**